### ISSUE TITLE
Comms error on basic install of 1.0.1

### ISSUE THREAD

**Issue by zmsl** [rossnichols/ABP-Naxx#2]
When testing addon in a simple raid, I am unable to get basic version check comms working. Version checks seem to work from the /fh start window, but when attempting to Sync I am getting the following error on the raid member that I'm attempting to sync to:

> 4H Assist: ERROR: Received an invalid addon comm from [char name]!

I dug into the lua and was able to enable verbose logging and found this info when triggering the encounter sync from the raid leader WoW client:

![lead_sync_error](https://user-images.githubusercontent.com/13343767/103981075-0ef2ee00-514f-11eb-95e2-cf280d59b71e.png)

but on the other client I'm not seeing much verbosity, only the same failure.

![client_sync_error](https://user-images.githubusercontent.com/13343767/103981072-0e5a5780-514f-11eb-842d-250c701fbc85.png)

I tried debugging further and ended up observing Ace Deserialize call not returning correctly (or at least my rudimentary print statements suggested that). Is there an exception thrown from Deserialize that I'm not seeing? Not sure at this point, and it's too late to continue

**Comment by zmsl** [comment]
Once I updated my addons (including DBM) the problem disappeared. Also, the changelog for DBM seems to relate to the Serializer problem area that I observed last night. Coincidence?! 
![2021-01-08_10-17-37](https://user-images.githubusercontent.com/13343767/104031605-e8f33b00-519a-11eb-84cd-ccebe8c1a002.png)

I'll verify that this issue is resolved with some of my guild members who were having the problem last night and close the issue if everything is OK.
